bot/exts/core/help.py

```diff
@@ -286,8 +286,9 @@ def _get_command_params(self, cmd: Command) -> str:
             # if required
             else:
                 results.append(f"<{name}>")
-
-        return f"{cmd.qualified_name} {' '.join(results)}"
+        if results:
```

Comment:
This can be simplified to
```py
return ' '.join([cmd.qualified_name, *results])
```

Reply:
Actually thats right thanks for this comment